### PR TITLE
docs: roadmap

### PR DESCRIPTION
**Pull request by sumo-drosiek**
##### Description

Document project roadmap

---

##### Checklist

- [ ] Changelog updated

###### Testing performed

- [ ] Redeploy fluentd and fluentd-events pods
- [ ] Confirm events, logs, and metrics are coming in

### CODE REVIEW

File: deploy/docs/roadmap.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Sumologic Kubernetes Collection Roadmap
+
+## [Helm Chart v2][v2]
+
+This version focuses on incorporating OpenTelemetry Collector as alternatives for the following elements:
```
Comment (sumo-drosiek) [inline deploy/docs/roadmap.md:5]:
```suggestion
This version focuses on incorporating OpenTelemetry Collector as alternative for the following elements:
```

File: deploy/docs/roadmap.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+We are also aiming to use OpenTelemetry Collector as alternative to Prometheus.
+
+## Helm Chart v4
+
+Fluent Bit and Fluentd are going to be removed and no longer used.
+
+Prometheus is going to be deprecated and replaced with OpenTelemetry Collector by default.
+
+We are also going to move metadata enrichment closer to it's sources,
+e.g. for logs it's going to be performed by daemonset (node agent)
```
Comment (sumo-drosiek) [inline deploy/docs/roadmap.md:30]:
```suggestion
e.g. for logs it's going to be performed by OpenTelemetry Collector daemonset (node agent)
```

File: deploy/docs/roadmap.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+- events collection
+- metadata enrichment for logs and metrics
+
+## [Helm Chart v3][v3]
+
+In this version Opentelemetry is going to be default choice for the following elements:
+
+- logs collection
+- events collection
+- metadata enrichment for logs and metrics
```
Comment (perk-sumo) [inline deploy/docs/roadmap.md:17]:
```suggestion
- metadata enrichment for logs, events and metrics
```

**Review by perk-sumo — APPROVED** [review]
LGTM, thank you @sumo-drosiek!

**Pull request merged**